src/rules/semicolonRule.ts

```diff
@@ -121,7 +121,9 @@ class SemicolonWalker extends Lint.RuleWalker {
 
     public visitPropertyDeclaration(node: ts.PropertyDeclaration) {
         const initializer = node.initializer;
-        if (initializer && initializer.kind === ts.SyntaxKind.ArrowFunction) {
+
+        // check if this is a multi-line arrow function (`[^]` in the regex matches all characters including CR & LF)
+        if (initializer && initializer.kind === ts.SyntaxKind.ArrowFunction && /\{[^]*\n/.test(node.getText())) {
```

Comment:
this condition is getting pretty long; I think the last test with the regex belongs in its own simple `function`; that would allow you to move the comment elsewhere as well